release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2419,6 +2420,10 @@ openshift.io/scc: hostmount-anyuid
 +
 (link:https://issues.redhat.com/browse/KATA-1373[*KATA-1373*])
 
+* There is currently a known issue that degrades the Machine Config Operator (MCO) when you attempt to create a cluster with both FIPS and `realTimeKernel`. As a workaround, modify the merge logic, which will prevent the MCO from degrading. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2096496[*BZ#2096496*])
+
+* The pipeline metrics API does not support the required `pipelinerun/taskrun` histogram values from OSP 1.6 and beyond. Consequently, the *Metrics* tab in the *Pipeline* -> *Details* page is removed instead of displaying incorrect values. There is currently no workaround for this issue. (link: https://bugzilla.redhat.com/show_bug.cgi?id=2074767[*BZ#2074767*])
```

Comment:
Does OSP refer to OpenStack Platform? If so, you could replace "OSP" with "{rh-openstack}".